sorc/ncep_post.fd/UPP_PHYSICS.f

```diff
@@ -4497,6 +4500,483 @@ SUBROUTINE CALSLR_UUTAH(SLR)
       END SUBROUTINE CALSLR_UUTAH
 !
 !-------------------------------------------------------------------------------------
+!
+!> Computes streamfunction and velocity potential from absolute vorticity
```

Comment:
@KarinaAsmar-NOAA Since this is a new subroutine incorporated in UPP, you might  reformat the fortran code provided by @JesseMeng-NOAA and @GeorgeVandenberghe-NOAA  readable and consistent as:

- all lowercase
- indentation for loops, conditionals and blocks